03-functions-and-loops/invest.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def invest(amount, rate, num_of_years):
+    initial_amount = amount
```

Comment:
Make `initial_amount` the parameter for the function and `amount` the computed variable inside the function. In general, we prefer to use a local variable for calculations and not the received parameter (unless you explicitly need to change the received parameter).